.github/workflows/playwright-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: playwright (integration test)
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'visual-js/visual-playwright/**'
+      - .github/workflows/js-build.yml
```

Comment:
Is that a left-over or an actual prerequisite ? I guess the 2nd, but prefer to be sure.

Reply:
Um, not a prerequisite per se, but changing of the JS Build system probably means we should run the integration tests for the affected tooling right?

Reply:
True 👍